docs/_data/mesheryctlcommands/lifecycle/system/status/command/status.yaml

```diff
@@ -0,0 +1,4 @@
+name: status
+description: Check status of Meshery and Meshery adapters.
+usage: |
+    mesheryctl system status [flags]
```

Comment:
Does this command have any flags?

Reply:
As far as concerned, this command doesn't have any flags. That's the reason I didn't review and approve it yet.

Reply:
This command doesn't have any flag for it but has the universal one -h, --help
As @alphaX86 had said, we can remove the [flag] param